test/parallel/test-tls-client-default-ciphers.js

```diff
@@ -40,7 +40,7 @@ function test1() {
   };
 
   try {
-    tls.connect(common.PORT);
+    tls.connect(0);
```

Comment:
Does it actually make sense to connect to port 0? Binding to port 0 allocates a free port but connecting to port 0 is not allowed in many programs. I guess because it is an invalid port so I'm not sure if it's ok here.